### ISSUE TITLE
Training time and async/await

### ISSUE THREAD

**Issue by talvasconcelos** [edwardhdlu/q-trader#9]
What is the python training time for this? With default settings 10 window and 1000 episodes, on what hardware?

Also, in Python is the fit() sequential? i mean, the code stops until model.fit() is done?

I'm trying to port this to tfjs and the code rushes through the for loop of the episodes and  the data length but doesn't "wait" for the model.fit(). Now i'm using async/await for it to wait for the model.fit() but it's taking a great amount of time. Can someone shed some light?

**Comment by kkuette** [comment]
This implementation is very slow... If you're using tensorflow i recommend you to see [tensorforce](https://github.com/reinforceio/tensorforce) for a faster training.

I can also suggest you to see my project called [TradzQAI](https://github.com/kkuette/TradzQAI).

**Comment by rui-ren** [comment]
Yes, this implementation is very slow, I have RTX 2060 and cunn, 1 hour can only train 10 episode .
